packages/flutter_tools/lib/src/asset.dart

```diff
@@ -770,3 +774,10 @@ _Asset _resolvePackageAsset(Uri assetUri, PackageConfig packageConfig) {
   globals.printError('Could not resolve package for asset $assetUri.\n');
   return null;
 }
+
+bool _isFileHidden(String relativeAssetFilePath){
```

Comment:
Pass in `fileSystem` and `platform` here, then make the method public and `@visibleForTesting`.

Reply:
(Instead of using `globals`)

Reply:
done!